aix/README.md

```diff
@@ -14,19 +14,19 @@ To build an AIX package, you need to download this repository and use the `gener
     $ cd aix
     ```
 
-2. Execute the `generate_rpm_package.sh` script to build the package. There are multiple parameters to select which package is going to be built, its architecture, etc. Here you can see all the different parameters:
+2. Execute the `generate_wazuh_packages.sh` script to build the package. There are multiple parameters to select which package is going to be built, its architecture, etc. Here you can see all the different parameters:
```

Comment:
Wrong script name

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/219/commits/136c1c0f1ca215fcb68503ddf7160fa47794df00